### PR TITLE
fix typos in README

### PR DESCRIPTION
**Pull request by giacom0rovers1**
Fixed minor typos in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ The form of specifications is inspired by  https://github.com/cf-convention/cf-c
 AsciiDoc format is a plaintext format which is easy to write and easy to parse. There are many tools to convert it to other formats such as html or pdf and GitHub can automaticaly preview it. 
 
 ## Usage
-Preview adoc files on gitHub webpagesite or fork it/download it and open it in any plain text editor.
+Preview adoc files on GitHub webpagesite or fork it/download it and open it in any plain text editor.
```
Comment (cchwala) [inline README.md]: "webpagesite" could also be fixed

**Review by cchwala — CHANGES_REQUESTED** [review]

**Comment by cchwala** [comment]
Thanks for these fixes @giacom0rovers1 

Maybe you can quickly check if there are some other typos. I found one, see my comment in the code. Would be good to catch as many as possible with this PR.

**Comment by cchwala** [comment]
Thanks for finding some more errors 🔍

**Comment by giacom0rovers1** [comment]
You're welcome :) happy to contribute

**Pull request merged**